drivers/flash/nrf_qspi_nor.c

```diff
@@ -487,16 +487,19 @@ static inline int qspi_nor_read_id(struct device *dev,
 static int qspi_nor_read(struct device *dev, off_t addr, void *dest,
 			 size_t size)
 {
+	void *dptr = dest;
```

Comment:
If this commit is cherry-picked from upstream then it should have a [nrf fromlist] tag.

Reply:
Will add